deploy/f2f-spec.yaml

```diff
@@ -20,6 +20,93 @@ tags:
 
 paths:
 
+  /callback:
+    post:
+      operationId: postCallback
+      summary: callback endpoint for asynchronous notifications from Yoti
+      description: >-
+        Endpoint is registered as the notifications callback with Yoti in the
+        call to `POST /session`
+      tags:
+        - Callback
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/CallbackRequest'
+      responses:
+        '202':
+          description: Accepted
+          headers:
+            Cache-Control:
+              schema:
+                type: string
+            Content-Type:
+              schema:
+                type: string
+            Strict-Transport-Security:
+              schema:
+                type: string
+            X-Content-Type-Options:
+              schema:
+                type: string
+            X-Frame-Options:
+              schema:
+                type: string
+        '401':
+          description: Unauthorized
+          headers:
+            Cache-Control:
+              schema:
+                type: string
+            Content-Type:
+              schema:
+                type: string
+            Strict-Transport-Security:
+              schema:
+                type: string
+            X-Content-Type-Options:
+              schema:
+                type: string
+            X-Frame-Options:
+              schema:
+                type: string
+        '500':
+          description: Internal Server Error
+          headers:
+            Cache-Control:
+              schema:
+                type: string
+            Content-Type:
+              schema:
+                type: string
+            Strict-Transport-Security:
+              schema:
+                type: string
+            X-Content-Type-Options:
+              schema:
+                type: string
+            X-Frame-Options:
+              schema:
+                type: string
+      x-amazon-apigateway-request-validator: both
```

Comment:
So I'd like to see some tests in place to see how this behaves when an invalid payload is set to this endpoint.